index.html

```diff
@@ -39,7 +39,7 @@ <h1>The Selenium Browser Automation Project</h1>
 
 driver = webdriver.Firefox()
 driver.get("http://google.com/?hl=en")
-search_box = driver.find_element_by_id("q")
+search_box = driver.find_element_by_id("lst-ib")
```

Comment:
`find_element_by_name("q")` is more stable.

Reply:
But it doesn't work

Reply:
It might be, yes. Do you want me to make that change and submit a fresh PR, closing this one?

Reply:
No need to create a new PR, just add another commit to your branch. It will update the PR